state/machine.go

```diff
@@ -2228,7 +2228,7 @@ func (m *Machine) CompleteUnitUpgradeSeries() error {
 		if err != nil {
 			return nil, err
 		}
-		if lock.CompleteStatus != model.CompleteStarted {
+		if lock.PrepareStatus != model.CompleteStarted {
```

Comment:
Given that we only have 1 Status now, don't we want to change it to name it something other than PrepareStatus?
'machine-status' is fine, but I would expect it would map to MachineStatus?